test/scripts/integration/README.md

```
@@ -0,0 +1,109 @@
+| Argument | Default | Description |
+|----------|---------|-------------|
+| `--url` | `http://localhost:40114` | Olla base URL |
+| `--timeout` | `30` | Request timeout in seconds |
+| `--skip-streaming` | off | Skip streaming tests |
+| `--skip-anthropic` | off | Skip Anthropic translator tests |
+| `--skip-providers` | off | Skip provider-specific route tests |
+| `--verbose` | off | Show response bodies |
+
+## Example Output
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)